src/device.py

```diff
@@ -61,22 +65,53 @@ def find_device_directory(self):
             self.error_message = None
         return found_directory
 
+    def deployToCPX(self):
+        device_directory = self.find_device_directory()
+        if self.error_message:
+            print(
+                "{}:\t{}".format(self.error_message[0], self.error_message[1]),
+                file=sys.stderr,
+                flush=True,
+            )
+        if self.connected:
+            dest_path = os.path.join(
+                device_directory, self.file_path.rsplit(os.sep, 1)[-1]
+            )
+            shutil.copyfile(self.file_path, dest_path)
+            message = {"type": "complete"}
+        else:
+            message = {"type": "no-device"}
+        return message
 
-if __name__ == "__main__":
-    import shutil
+    def deployToMicrobit(self):
+        try:
+            # Temporarily redirecting stdout because there are some print statements in uflash library
+            fake_stdout = open(os.devnull, "w")
+            _stdout = sys.stdout
+            sys.stdout = fake_stdout
+            uflash.flash(path_to_python=self.file_path)
+            sys.stdout = _stdout
+            message = {"type": "complete"}
+        except RuntimeError:
+            message = {"type": "wrong-python-version"}
```

Comment:
_&ast;nit&ast;_
Judging from the message this seems to be _low_ python version. Wrong python version suggests to me that there are particular version that don't work (as opposed to all python versions < 3.3).

Reply:
Yeah, I agree. I was wondering if we should just remove it though because our extension is promoted as having Python3.7+ as a requirement.